common/presenters/person-to-summary-list-component.js

```diff
@@ -2,10 +2,18 @@ const i18n = require('../../config/i18n')
 const filters = require('../../config/nunjucks/filters')
 
 function mapIdentifier({ value, identifier_type: type }) {
+  const translationKey = `fields::${type}.label`
+  const hasHtmlTranslation = i18n.exists(`${translationKey}_html`)
+  const key = {}
+
+  if (hasHtmlTranslation) {
+    key.html = i18n.t(translationKey, { context: 'html' })
+  } else {
+    key.text = i18n.t(translationKey)
+  }
+
   return {
-    key: {
-      text: i18n.t(`fields::${type}.label`),
```

Comment:
I think you might just be able to change the `text` property to `html` to output HTML instead of text?

Reply:
Oh, I see, we don't always know if it has HTML. I'd be tempted to just change it to `html` to all of them to avoid the need for the extra logic and checking against the translations. Would that solve the issue?